docs/src/pages/customization/how-to-customize/how-to-customize.md

```diff
@@ -1,3 +1,7 @@
+---
+components: GlobalStyles
```

Comment:
Haha, smart, I actually wonder if you would have gone this far

Reply:
I don't follow. That's what we do for every component? 

I'm actually planning to warn/throw if a component has no demos mentioned.

Reply:
True, I'm simply glad you thought to add it here. I forgot this when I first review the PR.